### PR TITLE
Api documentation

### PR DESCRIPTION
**Pull request by cadenmyers13**

### CODE REVIEW

**Comment by bobleesj** [comment]
looks good - the API package rendering nicely??

**Comment by cadenmyers13** [comment]
> looks good - the API package rendering nicely??

no docstrings present in the source files so there's nothing to build

File: doc/source/api/diffpy.fourigui.rst [hunk 1]
```diff
@@ -14,17 +12,12 @@ Subpackages
...
-
-.. |module| replace:: diffpy.fourigui.example_submodule module
-
-.. automodule:: diffpy.fourigui.example_submodule
-    :members:
-    :undoc-members:
-    :show-inheritance:
+    :titlesonly:
+
+    diffpy.fourigui.dist
```
Comment (sbillinge) [inline doc/source/api/diffpy.fourigui.rst:17]: Please check this.  I doubt we want most of these in the documentation.  Also, let's check how they got here....was it a problem with the API process that is also affecting other packages or something that went wrong locally?

**Review by sbillinge — COMMENTED** [review]
please see inline.  We will need an issue created to make docstrings!  This can be a project for a newbie in the future, but we need to make an issue now so we don't forget.

File: doc/source/api/diffpy.fourigui.rst (hunk 1 above)
Comment (cadenmyers13) [reply doc/source/api/diffpy.fourigui.rst:17]: @sbillinge The API section of the cookiecutter generates documentation for all files (see screenshot, not all files are listed). Not quite sure how to handle this. I will discuss with others 
Screenshot 2024-10-08 at 11 00 18 AM

Comment (bobleesj) [reply doc/source/api/diffpy.fourigui.rst:17]: Just to double check:

Did you run the following? 

```
python ../../release-scripts/auto_api.py diffpy.fourigui ./src/diffpy/fourigui ./doc/source/api
```

Comment (bobleesj) [reply doc/source/api/diffpy.fourigui.rst:17]: I am afraid it is due to the relative path issue. If you have the same problem, let's figure this out.

**Pull request closed without merging**